node/impl/full/eth_trace.go

```diff
@@ -498,6 +499,13 @@ func traceEthCreate(env *environment, addr []int, et *types.ExecutionTrace) (*et
 		// So we don't try and include a sentinel "impossible bytecode"
 		// value (the value specified by EIP-3541).
 		output = []byte{0xFE}
+		if deployedCode != nil {
+			var getBytecodeReturn evm12.GetBytecodeReturn
+			if err := getBytecodeReturn.UnmarshalCBOR(bytes.NewReader(et.MsgRct.Return)); err == nil && getBytecodeReturn.Cid != nil {
```

Comment:
@Stebalien 

Is this correct ? Can I get the byte code from the return value of the receipt on the top level transaction ?

Reply:
How else can I get the `cid` of the code here ?